_data/companies.yml

```diff
@@ -13,6 +13,11 @@
   image_url: images/logo_bbm.png
   description: Operated by KMK Online as a stand-alone business with teams operating from Indonesia, Singapore, Canada, UAE and South Africa.
 
+- name: Binar Academy
+  url: https://www.binar.co.id
+  image_url: images/logo_binar.png
```

Comment:
Sure.. will do it soon..
Could you please add the rules about image size in README, so it could be a nice guidance to whom would like to add/update it..

Thank you..